lighthouse-core/report/html/renderer/util.js

```diff
@@ -62,24 +64,45 @@ class Util {
     if (!clone.configSettings.locale) {
       clone.configSettings.locale = 'en';
     }
-    Util.setNumberDateLocale(clone.configSettings.locale);
```

Comment:
these are just rearranged so backward compatibility stuff, report locale setting, and smooshing aren't all interleaved